### PR TITLE
feat: shared with me remove teams menu

### PR DESCRIPTION
**Pull request by Kneesal**
# Description

### <samp>🤖 Generated by Copilot at 2aeb052</samp>

Added team management and localization features to the journeys admin app. The `AddJourneyButton` and `OnboardingPanelContent` components now use the `useTeam` hook to check and set the user's active team. The `TeamSelect` component has a new option for shared journeys. The `ActiveJourneyList` component supports internationalization with the `react-i18next` library.

- [Link to Basecamp Todo](https://3.basecamp.com/3105655/buckets/33034379/todos/6345357229)
- [Link to Basecamp Todo](https://3.basecamp.com/3105655/buckets/33034379/todos/6289493832)

# How should this PR be QA Tested?

Please describe the QA tests that you ran to verify your changes. Please also note any relevant details for your test configuration.

- [ ] Shared with me is selectable in Teams Select
- [ ] Cannot create new journeys when Selected With me is active team

# Walkthrough

### <samp>🤖 Generated by Copilot at 2aeb052</samp>

*  Enable internationalization for text in `Typography` components using `t` function from `react-i18next` library ([link](https://github.com/JesusFilm/core/pull/1693/files?diff=unified&w=0#diff-ca407cd8cf4cb2ec7283aa96210592a166a24e1d92de67dee9bae0799c979f91L152-R155))
*  Import and use `useTeam` hook from `TeamProvider` component to access and set the active team in `AddJourneyButton` and `OnboardingPanelContent` components ([link](https://github.com/JesusFilm/core/pull/1693/files?diff=unified&w=0#diff-c8f025029bd04bc3bb7287ec05d0e66421b1b397c01e68097e71c84e55e64a18L5-R11), [link](https://github.com/JesusFilm/core/pull/1693/files?diff=unified&w=0#diff-8f3c072dbc64b3d7d0f0b3a04d069ae84dd5c25f7519b5f5f0bb94816cbd6ba3R18))
*  Conditionally render `AddJourneyButton` and `ContainedIconButton` components based on the `activeTeam` value to prevent journey creation without a team ID ([link](https://github.com/JesusFilm/core/pull/1693/files?diff=unified&w=0#diff-c8f025029bd04bc3bb7287ec05d0e66421b1b397c01e68097e71c84e55e64a18L22-R35), [link](https://github.com/JesusFilm/core/pull/1693/files?diff=unified&w=0#diff-8f3c072dbc64b3d7d0f0b3a04d069ae84dd5c25f7519b5f5f0bb94816cbd6ba3L76-R87))
*  Import and use `useTranslation` hook from `react-i18next` library in `TeamSelect` component ([link](https://github.com/JesusFilm/core/pull/1693/files?diff=unified&w=0#diff-d590d146adecfb00651739a5c231d8eca4ffcbf3fb7518782e1a565e2c462feaL8-R15))
*  Change the `value` prop of the `Select` component in `TeamSelect` component to use `Date.now()` as a fallback to avoid Material UI warning ([link](https://github.com/JesusFilm/core/pull/1693/files?diff=unified&w=0#diff-d590d146adecfb00651739a5c231d8eca4ffcbf3fb7518782e1a565e2c462feaL24-R29))
*  Fix syntax error by adding a missing curly brace in the `onChange` handler of the `Select` component in `TeamSelect` component ([link](https://github.com/JesusFilm/core/pull/1693/files?diff=unified&w=0#diff-d590d146adecfb00651739a5c231d8eca4ffcbf3fb7518782e1a565e2c462feaL31-R35))
*  Add a new `MenuItem` component with the value of `Date.now()` to the `Select` component in `TeamSelect` component to allow the user to select "Shared With Me" option ([link](https://github.com/JesusFilm/core/pull/1693/files?diff=unified&w=0#diff-d590d146adecfb00651739a5c231d8eca4ffcbf3fb7518782e1a565e2c462feaR70-R80))

### CODE REVIEW

File: apps/journeys-admin/src/components/JourneyList/ActiveJourneyList/AddJourneyButton/AddJourneyButton.tsx [hunk 1]
```diff
@@ -19,14 +20,18 @@ export function AddJourneyButton(): ReactElement {
...
+    <>
+      {activeTeam != null && (
+        <Button
+          variant="contained"
+          startIcon={<AddIcon />}
+          size="medium"
+          onClick={handleClick}
+          sx={{ mt: 3, alignSelf: 'center' }}
+        >
+          Create a Journey
```
Comment (tataihono) [inline apps/journeys-admin/src/components/JourneyList/ActiveJourneyList/AddJourneyButton/AddJourneyButton.tsx:32]: add t function

File: apps/journeys-admin/src/components/Team/TeamSelect/TeamSelect.tsx [hunk 2]
```diff
@@ -21,14 +24,14 @@ export function TeamSelect(): ReactElement {
       <FormControl variant="standard" sx={{ minWidth: 100 }}>
         <Select
           disabled={query.loading}
-          value={activeTeam?.id ?? ''}
-          onChange={(event) =>
+          value={activeTeam?.id ?? 'Shared With Me'}
```
Comment (tataihono) [inline apps/journeys-admin/src/components/Team/TeamSelect/TeamSelect.tsx:27]: this should remain the same

File: apps/journeys-admin/src/components/Team/TeamSelect/TeamSelect.tsx [hunk 3]
```diff
@@ -63,6 +66,17 @@ export function TeamSelect(): ReactElement {
               {team.title}
             </MenuItem>
           ))}
+          <Divider />
+          <MenuItem
+            value="Shared With Me"
```
Comment (tataihono) [inline apps/journeys-admin/src/components/Team/TeamSelect/TeamSelect.tsx:71]: the value should be ""

**Review by tataihono — CHANGES_REQUESTED** [review]

File: apps/journeys-admin/src/components/Team/TeamSelect/TeamSelect.tsx (hunk 2 above)
Comment (Kneesal) [reply apps/journeys-admin/src/components/Team/TeamSelect/TeamSelect.tsx:27]: if value is '', it produces this result: 

image

**Pull request merged**